pages/index.js

```diff
@@ -50,24 +55,20 @@ class Page extends React.Component {
           <link
             rel="icon"
             type="image/png"
-            href={
-              this.state.now.isFriday()
```

Comment:
This was inconsistent with the API, as there, the check is:
```javascript
!time.isFriday() && !time.isWeekend()
```

We shouldn't show a red fav icon during the weekend?